sys/include/ztimer.h

```diff
@@ -488,11 +491,43 @@ void ztimer_update_head_offset(ztimer_clock_t *clock);
 void ztimer_init(void);
 
 /* default ztimer virtual devices */
+/**
+ * @brief   Base ztimer for the microsecond clock
+ */
+extern ztimer_clock_t *const ZTIMER_USEC_BASE;
+
 /**
  * @brief   Default ztimer microsecond clock
  */
 extern ztimer_clock_t *const ZTIMER_USEC;
 
+/**
+ * @def     ZTIMER_USEC_BASE_FREQ
+ *
+ * @brief   The frequency of the base ztimer for ZTIMER_USEC
+ */
+#ifdef XTIMER_HZ
+#define ZTIMER_USEC_BASE_FREQ   (XTIMER_HZ)
+#else
```

Comment:
On the first glimpse, this looks like `xtimer` configuration is creeping into `ztimer`. But I don't have any idea how to implement this the *clean way*.

Reply:
resolved, moved them to `ztimer/include/config.h`